src/status_im/ui/screens/hardwallet/setup/styles.cljs

```diff
@@ -208,6 +206,9 @@
 (def waiting-indicator-container
   {:height 200})
 
+(def progress-bar-container
+  {:width 280})
```

Comment:
Magic number?

Reply:
@jeluard css entirely consists of magic numbers :) do you think it's worth putting it into variable with name like `progress-bar-container-width`? 
```clj


(def progress-bar-containe-width 280)

(def progress-bar-container
  {:width progress-bar-container-width})
```